docs/source/features/errors.md

```diff
@@ -43,11 +42,7 @@ The response will return:
 In addition to stacktraces, Apollo Server's exported errors specify a human-readable string in the `code` field of `extensions` that enables the client to perform corrective actions. In addition to improving the client experience, the `code` field allows the server to categorize errors. For example, an `AuthenticationError` sets the code to `UNAUTHENTICATED`, which enables the client to reauthenticate and would generally be ignored as a server anomaly.
 
 ```js line=4,15-17
```

Comment:
These highlighted lines should line up with the AuthenticationError and it's usage, so I think we want to keep them on separate lines even though it might not match Prettier's styling